applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -277,9 +277,19 @@ void KratosGeoMechanicsApplication::Register() {
     KRATOS_REGISTER_CONDITION("UPwLysmerAbsorbingCondition3D3N", mUPwLysmerAbsorbingCondition3D3N)
     KRATOS_REGISTER_CONDITION("UPwLysmerAbsorbingCondition3D4N", mUPwLysmerAbsorbingCondition3D4N)
 
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition2D2N", mTNormalFluxCondition2D2N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition2D3N", mTNormalFluxCondition2D3N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition2D4N", mTNormalFluxCondition2D4N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition2D5N", mTNormalFluxCondition2D5N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition3D3N", mTNormalFluxCondition3D3N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition3D4N", mTNormalFluxCondition3D4N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition3D6N", mTNormalFluxCondition3D6N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition3D8N", mTNormalFluxCondition3D8N)
+    KRATOS_REGISTER_CONDITION("TNormalFluxCondition3D9N", mTNormalFluxCondition3D9N)
+
     //Register Constitutive Laws
-    KRATOS_REGISTER_CONSTITUTIVE_LAW("BilinearCohesive3DLaw",           mBilinearCohesive3DLaw)
-    KRATOS_REGISTER_CONSTITUTIVE_LAW("BilinearCohesive2DLaw",           mBilinearCohesive2DLaw)
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("BilinearCohesive3DLaw", mBilinearCohesive3DLaw)
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("BilinearCohesive2DLaw", mBilinearCohesive2DLaw)
```

Comment:
I think here we start having the Geo_ prefix for the conditions for thermal conditions to avoid future clashes with other applications, is that correct @avdg81?

Reply:
Yes, I would propose to adopt that naming convention here as well (i.e. letting the condition name start with `Geo`). Just to reduce the possibility of name clashes. When you make the change, please don't forget to update the test input files as well.

Reply:
Added Geo prefix to TNormalFluxCondition and TCondition